framework/auth/views.py

```diff
@@ -150,7 +154,8 @@ def auth_login(auth, **kwargs):
 
     status_message = request.args.get('status', '')
     if status_message == 'expired':
-        status.push_status_message('The private link you used is expired.')
+        status.push_status_message('The private link you used is expired.  Please <a href="/settings/account/"> +'
```

Comment:
Can we reduce a step for the user here and resend the link when they click "resend email" rather than directing them to the account settings?

This is nbd, though, you can add this in a followup PR if you want.